earthfile2llb/parser/EarthParser.g4

```diff
@@ -44,6 +45,8 @@ fromStmt: FROM (WS stmtWords)?;
 
 fromDockerfileStmt: FROM_DOCKERFILE (WS stmtWords)?;
 
+locallyStmt: LOCALLY WS*;
```

Comment:
We should include `(WS stmtWords)?` here.

I know it doesn't take arguments, but the errors that we produce from `listener` are usually more informative, in case someone does end up adding arguments.

Reply:
I was wondering if that was needed or not, i'll add it in.